hooks/hostname.hook/README.md

```diff
@@ -1,49 +1,45 @@
 ## Sample counting hook
```

Comment:
While we're editing this file, can we change this to say:
## Sample hostname assignment hook